### PR TITLE
Add QuickThreads, a vend for various clothes

### PR DESCRIPTION
**Pull request by Skarletto**
## About the PR

Makes drip more accessible with a vending machine
Kills normal jackboots storage and adds more combat boots in the secvend
Makes jamjar glasses description not sus
## Why / Balance

:3
## Technical details

much swag much YML
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/cosmatic-drift-14/cosmatic-drift/assets/122584947/ae0164a6-35a5-4b08-9c6f-5a053652b458

**Changelog**

:cl: Skarlet
- add: Epic new vend

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@
   parent: ClothingEyesBase
   id: ClothingEyesGlassesJamjar
   name: jamjar glasses
-  description: Also known as Virginity Protectors.
```
Comment (LankLTE) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:87]: 1984

File: Resources/Prototypes/Entities/Clothing/Shoes/boots.yml
[the remark this answers is not in the document]
Comment (LankLTE) [reply Resources/Prototypes/Entities/Clothing/Shoes/boots.yml:1]: You spawn with combat boots which can still hold stuff

[the remark this answers is not in the document]
Comment (Skarletto) [reply Resources/Prototypes/Entities/Clothing/Shoes/boots.yml:1]: Combat boots are untouched, all sec starts with combat boots. Unless you meant you just swap yours out?

File: Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml [hunk 2]
```diff
@@ -1358,6 +1358,42 @@
...
+  - type: VendingMachine
+    pack: QuickThreadsInventory
+    dispenseOnHitChance: 0.25
+    dispenseOnHitThreshold: 2
+    offState: off
+    brokenState: broken
+    normalState: normal-unshaded
+    ejectState: eject-unshaded
+    denyState: deny-unshaded
+    screenState: screen
```
Comment (LankLTE) [inline Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml:1376]:
```suggestion

```
idk the point of this but it's what's causing the fail

**Review by LankLTE — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml (hunk 2 above)
Comment (Skarletto) [reply Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml]: i copied the coffee machine prototype which has a screen state L

**Review by LankLTE — APPROVED** [review]

**Pull request merged**